rekall/plugins/addrspaces/amd64.py

```diff
@@ -105,6 +105,39 @@ def get_one_gig_paddr(self, vaddr, pdpte):
 
     lock = 0
 
+    def vaddr_access(self, vaddr):
+        """Is the access bit set on the page for the vaddr?"""
```

Comment:
this will be difficult to use since the return value varies in length. Maybe pad out to 4 values with None?